libs/shared/db/mysql/account/src/lib/cart.ts

```diff
@@ -28,16 +27,7 @@ export class Cart extends BaseModel {
   amount!: number;
   readonly version!: number;
 
-  static relationMappings = {
-    account: {
-      join: {
-        from: 'carts.uid',
-        to: 'accounts.uid',
-      },
-      modelClass: Account,
-      relation: BaseModel.BelongsToOneRelation,
-    },
-  };
+  static relationMappings = {};
 
```

Comment:
Will be dropping the rest of the objection related classes when I refactor the cart manager to use kysley.